F/fastp/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "fastp"
+version = v"0.23.3"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/OpenGene/fastp.git", "ca559a71feed94e74ea449e7567d0506de48dea4")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/fastp
+make -j${nproc}
+install -Dvm 755 "fastp${exeext}" "${bindir}/fastp${exeext}"
+mkdir -p "${prefix}/share/licenses/fastp/"
+cp ./LICENSE "${prefix}/share/licenses/fastp/"
```

Comment:
We have a utility to simplify this: https://docs.binarybuilder.org/stable/build_tips/#Installing-the-license-file
```suggestion
install_license LICENSE
```